dhctl/pkg/kubernetes/actions/manifests/manifests.go

```diff
@@ -348,7 +348,8 @@ func DeckhouseNamespace(name string) *apiv1.Namespace {
 			Name: name,
 			Labels: map[string]string{
 				"heritage": "deckhouse",
-				"extended-monitoring.deckhouse.io/enabled": "",
+				"extended-monitoring.deckhouse.io/enabled":          "",
+				"linstor-scheduler-admission.deckhouse.io/disabled": "",
```

Comment:
Won't it be applied on existing cluster configuration?

Reply:
It won't